ramls/marccat.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title:  MARCcat
-version: v1.2
+version: v1.5
```

Comment:
another big jump from 1.2 -> 1.5... what's the reason for this?

Reply:
I have changed it.